config/boot.rb

```diff
@@ -1,4 +1,14 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+
+# Note: 2019-02-23: In ruby 2.5.1 and 2.5.2 (fixed in 2.5.3)
+# IDE debugger (RubyMine) will not stop at breakpoints in Controller classes
+# unless you comment out the 'require' line for it in config/boot.rb
+#
+# Bug in ruby: https://bugs.ruby-lang.org/issues/14702
+#   see discussion in rails/rails for more info and this workaround:
+#      https://github.com/rails/rails/issues/32737#issuecomment-388763226
+#
```

Comment:
What's being commented out here?

Reply:
Line #8 says:
> # unless you comment out the 'require' line for it in config/boot.rb

In some IDEs (including RubyMine), you have to comment out line #14  if you are debugging with breakpoints in any Controller class.

Reply:
_TIL:_ If you copy a comment statement including the `#` Github markdown will interpret that `#` as a marker for *H1* and it will look like you are yelling.